torch/distributed/_composable_state.py

```diff
@@ -29,4 +29,7 @@ def _get_module_state(module: nn.Module) -> Optional[_State]:
     if isinstance(module, _State):
         return cast(_State, module)
     else:
-        return _module_state_mapping.get(module, None)
+        if module in _module_state_mapping:
+            return _module_state_mapping[module]
+        else:
+            return None
```

Comment:
get w/ default - NYI

Reply:
this one LGTM

Reply:
The new code is worse perf though, you do the dict lookup twice, whereas with `.get()` you only need to do it once